data-tool/scripts/README_COLIN_Corps_Extract.md

```diff
@@ -30,8 +47,11 @@ Transfer using 4 thread(s) filing_user ...                          5 rows in 00
 Transfer using 4 thread(s) office ...                               5 rows in 00:01. Reader waited 00:00, writer 00:04.
 ...
 ```
-8. Re-enable triggers in target extract db using sql snippet found in `misc_extract_corps_queries.sql`
+8. (No longer required) ~~Re-enable triggers in target extract db using sql snippet found in `misc_extract_corps_queries.sql`~~
 9. Re-index target extract db.
+10. Use "count overview" SQL snippet in `misc_extract_corps_queries.sql` to verify the db changes.
```

Comment:
ok, i guess it does do something :)  

we can just remove the steps(6 & 8) from this readme i guess.